rcsb/utils/dictionary/DictMethodEntryHelper.py

```diff
@@ -762,27 +774,28 @@ def addEntryInfo(self, dataContainer, catName, **kwargs):
                 xObj = dataContainer.getObj("struct")
                 if xObj.hasAttribute("pdbx_structure_determination_methodology"):
                     methodType = xObj.getValue("pdbx_structure_determination_methodology", 0)
+            if not methodType:
+                if dataContainer.exists("exptl"):
```

Comment:
Are we positive all experimental models in the archive will have `exptl`? (Even older ones)?

Reply:
I think so.